hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -1009,8 +992,6 @@ class Emit[C](
                       (lStackVars :+ n :+ k, rStackVars :+ k :+ m)
                   }
 
-                  val lElem = leftPVal.loadElement(lIndices, cb)
-                  val rElem = rightPVal.loadElement(rIndices, cb)
```

Comment:
yeah like here :)

Reply:
Yeah exactly. I think I found all the relevant ones to this change though, seems to pass all tests now.